src/sidebar/components/search-input.js

```diff
@@ -29,7 +29,10 @@ export default function SearchInput({ alwaysExpanded, query, onSearch }) {
 
   const onSubmit = e => {
     e.preventDefault();
-    onSearch(input.current.value);
+    // Don't submit an empty query
+    if (input.current.value.trim()) {
```

Comment:
This now became an issue because just clicking the search icon would normally invoke the search. Prior to this change, it didn't cause any side effects because the selection was not being cleared. But now that simple action will clear out any selected annotation -- which a user wouldn't want unless they were actually making a search. The simple fix is to confirm there is something in the input before firing off the search event.